roles/openshift_cloud_provider/templates/openstack.conf.j2

```diff
@@ -2,11 +2,6 @@
 auth-url = {{ openshift_cloudprovider_openstack_auth_url }}
 username = {{ openshift_cloudprovider_openstack_username }}
 password = {{ openshift_cloudprovider_openstack_password }}
-{% if openshift_cloudprovider_openstack_domain_id is defined %}
-domain-id = {{ openshift_cloudprovider_openstack_domain_id }}
-{% elif openshift_cloudprovider_openstack_domain_name is defined %}
-domain-name = {{ openshift_cloudprovider_openstack_domain_name }}
-{% endif %}
```

Comment:
we'll probably want to attempt to keep the domain_id and domain_name settings here.